.github/workflows/ci-release-2020.06.yml

```diff
@@ -3,19 +3,18 @@ name: Garage CI Release-2020.06
 env:
   GARAGE_GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
   CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
-  TRAVIS_COMMIT_RANGE: origin/${{ github.base_ref }}...origin/${{ github.head_ref }}
+  CI_COMMIT_RANGE: origin/${{ github.base_ref }}...origin/${{ github.head_ref }}
```

Comment:
I think we should call it PR_COMMIT_RANGE because this value doesn't make sense for any other triggers, and also is empty probably